manifest.json

```diff
@@ -1,24 +1,33 @@
 {
   "name": "Markdown Link",
-  "version": "0.2.1",
-  "description": "Copy Markdown link for current tab.",
+  "version": "0.3.0",
+  "description": "Copy Markdown link for current tab",
   "icons": {
     "128": "images/icon-128.png"
   },
-  "permissions": ["activeTab", "clipboardWrite"],
-    "background": {
-      "scripts": ["background.js"],
-      "persistent": false
-    },
-  "browser_action": {
+  "permissions": [
+    "activeTab",
+    "clipboardWrite",
+    "downloads",
+    "scripting",
+    "storage"
+  ],
+  "background": {
+    "service_worker": "background.js"
+  },
+  "options_ui": {
+    "page": "options.html"
+  },
+  "action": {
     "default_title": "Get Markdown URL"
   },
   "commands": {
-    "_execute_browser_action": {
+    "getCurrentLink": {
```

Comment:
Can we use the original command ID? :D

Reply:
It genereates the following error:
![image](https://github.com/wdhongtw/markdown-link/assets/17429937/8766e70d-3d75-4d99-8f3e-93e22f288dfd)

Reply:
I think it is must be _execute_action in manifest V3
https://developer.chrome.com/docs/extensions/reference/commands/#action-commands